internal/handlers/assisted_service_client.go

```diff
@@ -91,9 +91,9 @@ func (c *AssistedServiceClient) ramdiskContent(imageServiceRequest *http.Request
 // ignitionContent returns the ignition data on success and the error and the corresponding http status code
 // The code is also returned to ensure issues with authentication from the assisted service request are communicated back to the image service user
 // The returned code should only be used if an error is also returned
-func (c *AssistedServiceClient) ignitionContent(imageServiceRequest *http.Request, imageID string) (*isoeditor.IgnitionContent, int, error) {
+func (c *AssistedServiceClient) ignitionContent(imageServiceRequest *http.Request, imageID string) (*isoeditor.IgnitionContent, string, int, error) {
```

Comment:
I don't like all these return values.
If someone has a better idea for how to organize this I'd be happy for a suggestion.